media-libs/libgroove/files/libgroove-4.3.0_cflags.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
reference the upstream commit messages with an URL here and give a short summary like "remove -Werror from compiler flags"